.github/workflows/tests.yml

```diff
@@ -3,11 +3,16 @@ name: spotify-dl tests
 on: [push, pull_request]
 
 jobs:
+  deprecated:
+    name: build (3.6)
+    runs-on: ubuntu-latest
+    steps:
+      - run: 'echo "Python 3.6 is not supported anymore, no build required"'
```

Comment:
I think we can just get rid of this

Reply:
Ok, but that will leave the CI with an expected check status, I though it would be better to merge the PR first & then get rid of this as you suggest.

Reply:
the checks are associated with the job, I don't think I added them manually. So removing 3.6 from the matrix should remove the check automatically. If not, I can remove it from settings. Try removing, see what happens. Worst case scenario I can force merge, or we can rollback the changes